src/main.cpp

```diff
@@ -384,6 +385,12 @@ int main(int argc, char *argv[])
         }
     });
 
+    QTranslator trans, qttrans;
+    qttrans.load(QLocale(), QLatin1String("qt"), "_", "../share/qt5/translations");
```

Comment:
@Carmina16: fyi: this will work only if you are in the /usr/* on *nix and probably it won't work on windows at all. Just opened #194 which fixes that. :)